cmd/web.go

```diff
@@ -48,6 +48,26 @@ and it takes care of all the other things for you`,
 	},
 }
 
+func runHTTPRedirector() {
+	source := setting.HTTPAddr + ":80"
+	dest := strings.TrimSuffix(setting.AppURL, "/")
+	log.Info("Redirecting: %s to %s", source, dest)
+
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		target := dest + r.URL.Path
+		if len(r.URL.RawQuery) > 0 {
+			target += "?" + r.URL.RawQuery
+		}
+		http.Redirect(w, r, target, http.StatusTemporaryRedirect)
+	})
```

Comment:
Just take `r.URL` and change Schema to `https` 😕

Reply:
@bkcsoft your requested change seems to conflict with with the change I just made for @lunny?  Can you please re-visit and see what you now think?  Also I have an open question above that I was hoping you would comment on.